public/css/quizzard.css

```diff
@@ -3,3 +3,32 @@ body {
   padding-bottom: 20px;
   background-color: #eee;
 }
+
+#snackbar {
```

Comment:
Can we test this on smaller devices to make sure the fixed size will work

Reply:
Not sure how but this is a good idea. Can alway be adjusted later when we go back for nicer styling.